apps/www/_blog/2022-08-01-sharing-with-you-the-supabase-flutter-sdk-1-developer-preview.mdx

```diff
@@ -0,0 +1,120 @@
+
+```dart
+// Before
+final response = await Supabase.instance.from('messages').select().execute();
+final data = response.data;
+
+// After
+final data = await Supabase.instance.from('messages').select();
+```
+
+## Desktop support for deeplinks
+
+Ever since supabase-flutter was born, it supported only iOS, Android and Web for deep linking. This was a limitation of the deep link library that we were using, but with the 1.0 launch, we are moving to use [app_links](https://pub.dev/packages/app_links), which will enable us to support MacOS and Windows applications as well! Linux support is being worked on as well. 
+
+![Supabase Flutter desktop support](/images/blog/flutter-1/supported-platforms-table.png)
+
+
+## Multiplayer support
+
+[Multiplayer](https://supabase.com/blog/2022/04/01/supabase-realtime-with-multiplayer-features) is the next generation Supabase realtime engine that was announced at the previous launch week as coming out soon. We want our Flutter developers to experience this new multiplayer feature as well, so are working hard at bringing it to our Flutter SDK. It is not yet included in the developer preview of Supabase Flutter 1.0, but will be part of it when stable launch has been released. 
+
+## Supabase Auth UI for Supabase
+
+Last but not least, we are bringing you another library, the Supabase Auth UI for Supabase! When released, this library will enable you to implement a basic authentication screen without having to build it yourself. You can just load the library and display a nice looking auth ui. The library will take your theme settings automatically to match the look and feel of your application. You can easily enable/ disable certain auth features with the configuration as well. I would like to thank [Fatuma](https://twitter.com/XquisiteDreamer) for single handedly working on bringing us an easier authentication experience. 
+
+## Final thoughts
+
+The updates that I got to introduce here are just the tip of the iceberg, and there have been many bug fixes or other features constantly being added to the Supabase Flutter SDK. This could not have been possible with the help from the open source community. Here, I would also like to give a shout out to two other developers who have been a major part of the journey of this SDK, and they are [Vinzent](https://twitter.com/Vinzent03_) and [Daniel Mossaband](https://github.com/DanMossa). They have been a huge part of the Supabase Flutter SDK not just for the 1.0 release, but throughout the lifetime of the library. 
```

Comment:
```suggestion
These updates are just the tip of the iceberg for 1.0. There are been many bug fixes and features constantly being added to the Supabase Flutter SDK. This could not have been possible with the help from the open source community. Here, I would also like to give a shout out to two other developers who have been a major part of the journey of this SDK: [Vinzent](https://twitter.com/Vinzent03_) and [Daniel Mossaband](https://github.com/DanMossa). They have been a huge part of the Supabase Flutter SDK - not just for the 1.0 release, but throughout the lifetime of the library. 
```